index.html

```diff
@@ -1962,8 +1962,20 @@ <h3>Verify Proof</h3>
 integrity of a <a>secured data document</a> by verifying its
 digital proof. Required inputs are a
 <a>secured data document</a> (<var>securedDocument</var>) and
-<a>proof options</a> (<var>options</var>). A <dfn>verification result</dfn> is
-produced as output.
+<a>proof options</a> (<var>options</var>). The <a>proof options</a> MAY contain
+the following values that help protect against relay and replay attacks:
+<var>expectedProofPurpose</var>,
+<var>acceptableCreatedTimeDeviationInSeconds</var>, <var>domain</var>, and
```

Comment:
I'd prefer to see this renamed to something like `maxTimestampDelta` so it's both shorter and can be used for any time comparisons, both for `created` and `expires` generally.

Reply:
In fact, I recommend we remove being prescriptive about checking times in the algorithms and instead mention that implementations may provide interfaces to allow applications to use business rules to check the timeliness of proofs. Otherwise, we'll end up having more complex algorithms where the "current time" can be passed in and so on -- all checks that implementations can decide how to / not to expose information. What's important for our specs is that cryptosuites secure over these parameters so they are tamper-evident -- such that applications / implementations can check them as they see fit.

Reply:
Ok, I'll remove it in the next iteration.

Reply:
Removed in 4d802f7cd6bcaa039b1835d4ded917b1224bd30f.